Makefile

```diff
@@ -12,7 +12,9 @@ DOCKER_DEB_TEST := sh -euxc ' \
 	&& tmp=$$(mktemp -d) \
 	&& cp -r /mnt/* "$$tmp" \
 	&& cd "$$tmp" \
-	&& pip install pytest \
+	&& pip install --upgrade pip \
+	&& /usr/local/bin/pip install --upgrade setuptools distribute \
+	&& /usr/local/bin/pip install -r requirements-dev.txt \
```

Comment:
this is the only way I can get mock to install on an `ubuntu:lucid` Docker (ancient pip, setuptools, etc.)

I know it looks really hacky, but I tried everything else :\